contract/.gitignore

```
@@ -5,6 +5,7 @@ node_modules
 /cache
 /artifacts
 /ignition/deployments/chain-31337
+/ignition/deployments/chain-11155111/build-info
```

```suggestion
/ignition/deployments/*/build-info
```
maybe like this so it's ignoring every build-info, idk if it's double asterisk or just one asterisk